pilot/pkg/model/context.go

```diff
@@ -156,6 +157,7 @@ func DefaultProxyConfig() meshconfig.ProxyConfig {
 		ProxyAdminPort:         15000,
 		ControlPlaneAuthPolicy: meshconfig.AuthenticationPolicy_NONE,
 		CustomConfigFile:       "",
+		Concurrency:            int32(runtime.NumCPU()),
```

Comment:
Can you confirm this on a docker (or k8s) env, when running on a large machine but with 
a smaller allocation.  For example a 16 CPU machine, with docker set to use 1 CPU only (
AFAIK k8s works the same).

Reply:
@costinm This works as expected in a k8s environment, the runtime.NumCPU() returns the bare host CPU value.